src/object/obj_rpc.c

```diff
@@ -209,9 +154,6 @@ crt_proc_daos_iod_and_csum(crt_proc_t proc, crt_proc_op_t proc_op,
 	if (rc != 0)
 		return rc;
```

Comment:
Is there a reason this returns rc where the two cases below it returns -DER_HG

Reply:
Thanks. It looks I have to make them similar.